sdk/servicebus/service-bus/README.md

```diff
@@ -302,7 +302,7 @@ If you'd like to contribute to this library, please read the [contributing guide
 
 ![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-js%2Fsdk%2Fservicebus%2Fservice-bus%2FREADME.png)
 
-[apiref]: https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-service-bus/7.0.0-preview.2/index.html
+[apiref]: https://aka.ms/azsdk/js/service-bus/docs/7.0.0-preview.2
```

Comment:
You sure you need version number in the aka?

Reply:
This comes from my lack of understanding regarding how azuresdkdocs works. If I go to <https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-service-bus/> it gives me an error. What should the version-less URL be?

Reply:
I don't think there is a "latest" preview link.  @kaerm ?

Reply:
as far as i know there is no latest. The way it works is that the azure.github.io/azure-sdk-for-js is the index that points to different blobs for each package. So azure.github.io/azure-sdk-for-js should be the landing page

Reply:
That makes sense!Thank you!

Reply:
Do you mean this? https://azure.github.io/azure-sdk-for-js/identity.html this will let them pick the version. I'll use that one for now.

Reply:
I've done this preemptively. Hoping this help us save some time.

Reply:
Exactly, awesome!